### ISSUE TITLE
Installer needs to check for MSVCP140.dll presence

### ISSUE THREAD

**Issue by Ambrose2018** [gitahead/gitahead#190]
I have installed the software two different ways and encountered the same problem. The first attempt to install was via the official download (GitAhead-win64-2.5.5.exe) and later after uninstalling that I tried using Chocolatey (same GitAhead version) since choco sometimes catches dependency issues. In both cases the install completed successfully with no errors but when I attempted to run the program it aborted with the following alert dialog: 
![#_6068](https://user-images.githubusercontent.com/48845067/55673447-163fe600-585d-11e9-84b2-6396e3ce2ee9.jpg)

I was able to fix the problem by getting the copy of the needed DLL from my Java SE8 install (...\JDK\tools\jre\bin\msvcp140.dll --- file version 14.10.25008.0 dated 12/19/2017) and placing it into the executable folder for GitAhead). 

NOTE -- I mention the version above because I tried to fix the problem using a more recent version of this DLL and GitAhead failed. So it appears GitAhead is somewhat version dependent with regards to the VC runtime. Separate issue probably.

**Comment by hackhaslam** [comment]
GitAhead includes the appropriate visual studio redistributable installer and runs it as part of the install process. It's called vcredist_x64.exe. It's visible in your screenshot. Does it fail when you try to run it manually?

**Comment by Ambrose2018** [comment]
No, the VC runtime installer does not fail, or at least GitAhead is not reacting to it if it does. As I said the installer completes without error. That being said I reinstalled GitAhead and watched its installer with a special eye to seeing what **vcredist_x64.exe** did. What I see is that it seemed to run a very long time compared to all the other steps of the GitAhead installer.

**Comment by hackhaslam** [comment]
Okay, so when you run it manually (double-click on vcredist_x64.exe in the install directory) it completes successfully without reporting any errors?

**Comment by Ambrose2018** [comment]
It looks like maybe the problem (for my computer anyways) is that I do not have KB2999226 installed. That is on me, but still the GitAhead installer should look for the path presence of **MSVCP140.dll** (and also **CONCRT140.dll** which may be called by the first DLL) rather than _assume_ the VC runtime installer did the job. 

I'm attaching the log file from my run of vcredist_x64 if it helps... 
[vcredist_x64.log](https://github.com/gitahead/gitahead/files/3051909/vcredist_x64.log)

**Comment by Ambrose2018** [comment]
Sorry, clicked the wrong button, did not mean to close. I have a workaround and a probable cause for my specific situation, but the GitAhead installer should have caught this which was the original reason for my opening this issue.
